
Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mcredential-initializer[0m
{"level":"info","ts":1614945141.2769115,"caller":"creds-init/main.go:44","msg":"Credentials initialized."}

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mworking-dir-initializer[0m

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mplace-tools[0m

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-niranjankolli-python-jinja2-log-ndk9r-jj6hl[0m
{"level":"info","ts":1614945158.4932213,"caller":"git/git.go:105","msg":"Successfully cloned https://github.com/Niranjankolli/python-jinja2-login.git @ v0.0.4 in path /workspace/source"}
{"level":"warn","ts":1614945158.4932954,"caller":"git/git.go:152","msg":"Unexpected error: creating symlink: symlink /tekton/home/.ssh /root/.ssh: file exists"}
{"level":"info","ts":1614945158.520497,"caller":"git/git.go:133","msg":"Successfully initialized and updated submodules in path /workspace/source"}

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-jx-git-credentials[0m
Generated Git credentials file /workspace/xdg_config/git/credentials

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mniranjankolli-python-jinja2-log-ndk9r-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
error checking push permissions -- make sure you entered the correct tag name, and that you are authenticated correctly, and try again: checking push permission for "https://hub.docker.com//*************/python-jinja2-login:0.0.4": creating push check transport for https: failed: Get "https://https/v2/": dial tcp: lookup https on 10.100.0.10:53: no such host
[31m
Pipeline failed on stage 'from-build-pack' : container 'step-build-container-build'. The execution of the pipeline has stopped.[0m
